source/tutorials/fmt-cdiff.rst

```diff
@@ -0,0 +1,478 @@
+The FastTree program creates an unrooted tree, so in the final step in this section midpoint rooting is applied to place the root of the tree at the midpoint of the longest tip-to-tip distance in the unrooted tree.
+
+.. command-block::
+
+   qiime phylogeny align-to-tree-mafft-fasttree \
+     --i-sequences rep-seqs.qza \
+     --o-alignment aligned-rep-seqs.qza \
+     --o-masked-alignment masked-aligned-rep-seqs.qza \
+     --o-tree unrooted-tree.qza \
+     --o-rooted-tree rooted-tree.qza
+
+
+.. _`fmt cdiff diversity`:
+
+Alpha and beta diversity analysis
+---------------------------------
+
+QIIME 2's diversity analyses are available through the ``q2-diversity`` plugin, which supports computing alpha and beta diversity metrics, applying related statistical tests, and generating interactive visualizations. We'll first apply the ``core-metrics-phylogenetic`` method, which rarefies a ``FeatureTable[Frequency]`` to a user-specified depth, computes several alpha and beta diversity metrics, and generates principle coordinates analysis (PCoA) plots using Emperor for each of the beta diversity metrics. The metrics computed by default are:
+
+* Alpha diversity
+
+  * Shannon's diversity index (a quantitative measure of community richness)
+  * Observed OTUs (a qualitative measure of community richness)
+  * Faith's Phylogenetic Diversity (a qualitiative measure of community richness that incorporates phylogenetic relationships between the features)
+  * Evenness (or Pielou's Evenness; a measure of community evenness)
+
+* Beta diversity
+
+  * Jaccard distance (a qualitative measure of community dissimilarity)
+  * Bray-Curtis distance (a quantitative measure of community dissimilarity)
+  * unweighted UniFrac distance (a qualitative measure of community dissimilarity that incorporates phylogenetic relationships between the features)
+  * weighted UniFrac distance (a quantitative measure of community dissimilarity that incorporates phylogenetic relationships between the features)
+
+An important parameter that needs to be provided to this script is ``--p-sampling-depth``, which is the even sampling (i.e. rarefaction) depth. Because most diversity metrics are sensitive to different sampling depths across different samples, this script will randomly subsample the counts from each sample to the value provided for this parameter. For example, if you provide ``--p-sampling-depth 500``, this step will subsample the counts in each sample without replacement so that each sample in the resulting table has a total count of 500. If the total count for any sample(s) are smaller than this value, those samples will be dropped from the diversity analysis. Choosing this value is tricky. We recommend making your choice by reviewing the information presented in the ``table.qzv`` file that was created above and choosing a value that is as high as possible (so you retain more sequences per sample) while excluding as few samples as possible.
```

Comment:
``table.qzv`` --> feature table summary (``table.qzv``)

Reply:
> Well, it depends on if you are using deblur or dada2, right?

The tutorial has to use one or the other (dada2 or deblur), as structured it isn't possible to use both (you would need to duplicate all of these commands).

Ultimately, the point was "why 500?" - it looks like you have addressed that. Thanks!